dev_guide/builds/build_inputs.adoc

```diff
@@ -196,6 +196,9 @@ specific commit, the system will fallback to a regular Git clone operation and
 checkout the commit, because using the `--depth=1` option only works with named
 branch refs.
 
+If the `ref` field denotes a pull request, the system will utilize a Git fetch operation
```

Comment:
@gabemontero Showing the user actual commands is best, so I suggest that we update the previous instance as well. You can do that in this PR, or I can follow-up in a separate PR. Thanks!